tests/test_dse.py

```diff
@@ -164,8 +164,8 @@ def test_tti_rewrite_aggressive(tti_nodse):
 
 @skipif_yask
 @pytest.mark.parametrize('kernel,space_order,expected', [
-    ('shifted', 8, 364), ('shifted', 16, 830),
-    ('centered', 8, 170), ('centered', 16, 306)
+    ('shifted', 8, 355), ('shifted', 16, 811),
+    ('centered', 8, 168), ('centered', 16, 300)
```

Comment:
better flop counts? 😮

Reply:
Yes, so the new method of substituting spacing symbols in the seismic stencils only captures space dimensions, not time as before. So `dt` is now a runtime argument, but there are only very few explicit multiplications with `dt` in the stencil, so they should not affect performance significantly.

Why it's going down instead of up, I don't know. Maybe the DSE finds better factorization/lifting opportunities if we don't pre-resolve `dt`?  Not sure though...

Reply:
or perhaps the baseline is lower